### ISSUE TITLE
Seeking explanation on attribute classification task loss

### ISSUE THREAD

**Issue by pbamotra** [fdjingyuan/Deep-Fashion-Analysis-ECCV2018#2]
Hi authors,

Can you please explain your implementation of loss for the attribute classification branch. I'm no able to comprehend the 2*1000 output layer and the use of cross entropy on that. That data is multi-label classification if I'm not wrong, how do you modify the cross entropy loss to use on multi-label data, it should've been Sigmoid+BCELoss if I'm following the online PyTorch tutorials correctly.

Thanks

**Comment by Gaumera** [comment]
hi,
I also have confusion about the 2*1000 output size while data have 1000 attributes itself. can you please explain about this issue?
